src/TabBar.js

```diff
@@ -25,7 +25,11 @@ class TabBar extends Component {
         `No action is defined for name=${el.props.name} ` +
         `actions: ${JSON.stringify(Object.keys(Actions))}`);
     }
-    Actions[el.props.name]();
+    if (el.props.onPress){
```

Comment:
can you make sure it is a function sent ( typeof el.props.onPress === "function")